src/tests/server.mjs

```diff
@@ -23,6 +23,20 @@ router.use((request, response, next) => {
   }
 })
 
+router.get('/frames', (req, res) => {
+  // The purpose of this route is to do a server-side replacement of ##randNum##. This way when we
+  // refresh this page we are guaranteed to see a change in content.
+  fs.readFile(path.join(__dirname, '../../src/tests/fixtures/frames.html'), 'utf8', (err, data) => {
+    if (err) {
+      console.error(err)
+      return res.status(500).send('Error reading file')
+    }
+    const randomNumber = Math.floor(Math.random() * 10000000)
+    const processedContent = data.replace('##randNum##', randomNumber.toString())
+    res.send(processedContent)
+  })
+})
+
```

Comment:
The test suite has generally avoided purpose-built server behavior in favor of general endpoints that either render fixtures directly or redirect.

What is being exercised by replacing `##randNum##`? Are there other ways to cover the intended behavior without introducing a new server response?

Reply:
@seanpdoyle Yes, this is crufty. I didn't love it. I can safely remove this and all tests will still pass since the tests are just asserting that various events are firing.

I personally find it helpful to be able to actually observe with my own eyes with a morph is happening (i.e. loading localhost:9000 and opening inspector and watching which elements flash). Without a single element changing on reload I could not visually confirm that a morph was occurring and the pattern of our tests so far simply asserts that an event was fired. That's all this was for.

I'll go ahead and remove it.